msal/src/Microsoft.Identity.Client/IUser.cs

```diff
@@ -47,7 +47,7 @@ public interface IUser
         /// <summary>
         /// Gets identity provider if returned by the service. If not, the value is null.
         /// </summary>
-        string IdentityProvider { get; }
+        //string IdentityProvider { get; }
```

Comment:
>//string IdentityProvider { get; } [](start = 8, length = 34)

remove?

Reply:
will do after confirming exact list of claims to be returned as User -> Account

Reply:
ok

---
In reply to: [191035389](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1065#discussion_r191035389) [](ancestors = 191035389)